Client/Frontend/Browser/Tabs/InactiveTabs/InactiveTabsFooterView.swift

```diff
@@ -0,0 +1,95 @@
+
+import Common
+import UIKit
+
+class InactiveTabsFooterView: UICollectionReusableView, ReusableCell, ThemeApplicable {
+    struct UX {
+        static let imageSize: CGFloat = 29
+        static let borderViewMargin: CGFloat = 16
+        static let buttonInset: CGFloat = 14
+        static let buttonImagePadding: CGFloat = 11
+        static let buttonFontSize: CGFloat = 16
+        static let buttonCornerRadius: CGFloat = 13.5
+        static let buttonBorderWidth: CGFloat = 1
+    }
+    // MARK: - Properties
+    var buttonClosure: (() -> Void)?
+
+    let containerView = UIView()
+    var shouldLeftAlignTitle = false
+
+    // MARK: - UI Elements
+    private lazy var roundedButton: UIButton = {
+        let button = UIButton()
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
+            withTextStyle: .body,
+            size: UX.buttonFontSize)
+        button.setImage(UIImage(systemName: "trash"), for: .normal)
+        button.setTitle(.TabsTray.InactiveTabs.CloseAllInactiveTabsButton, for: .normal)
+        button.titleLabel?.textAlignment = .center
+        button.titleLabel?.lineBreakMode = .byTruncatingTail
+        button.layer.cornerRadius = UX.buttonCornerRadius
+        button.layer.borderWidth = UX.buttonBorderWidth
+        button.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    // MARK: - Initializers
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setupView() {
+        addSubview(roundedButton)
+
+        roundedButton.contentEdgeInsets = UIEdgeInsets(
+            top: UX.buttonInset,
+            left: UX.buttonInset,
+            bottom: UX.buttonInset,
+            right: UX.buttonInset)
+        roundedButton.titleEdgeInsets = UIEdgeInsets(
+            top: 0,
+            left: UX.buttonImagePadding,
+            bottom: 0,
+            right: UX.buttonImagePadding
+        )
+
+        let trailingOffSet: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 100 : 23
+        let leadingOffSet: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 100 : 23
+
+        NSLayoutConstraint.activate([
+            roundedButton.centerXAnchor.constraint(equalTo: centerXAnchor),
+            roundedButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: trailingOffSet),
+            roundedButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -leadingOffSet),
+            roundedButton.topAnchor.constraint(equalTo: topAnchor, constant: 16),
+            roundedButton.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -24)
+        ])
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+    }
```

Comment:
What is the purpose of overriding this?

Reply:
copy-paste the declaration from Legacy tableView footer but I haven't added the code yet. I will add it I cutoff the PR afraid of passing the big PR warning but it seems I'm good for a few more lines 😄